1.0/tools/tests.md

```diff
@@ -59,9 +59,9 @@ Follow the steps below to get set up, or watch the Polycast:
 
         npm install -g web-component-tester
 
-    On Mac OS X you need to [manually install][selenium] the latest Safari 
-    extension for Selenium. See the 
-    [Web Component Tester Polycast][workaround-example] for 
+    On Mac OS X you need to [manually install][selenium] the latest SafariDriver
```

Comment:
@tony19 I think the change is good. The wording here is a little confusing, but that's our fault. 

I'm a little iffy about linking to the current version of the driver, because we're unlikely to notice when that version changes. However, without the direct link it's nearly impossible to find, because currently it's _only_ in the 2.48 folder, not in any of the four subsequent releases.

Thoughts on this @robdodson? Inclined to leave the link for now, and work on the wording here later.